docs/firebase.md

```diff
@@ -230,3 +230,6 @@ DAQ -> Firebase: Merged (active)
 Firebase -> WebApp: Merged (active)
 note right of WebApp: "Normal" state
 ```
+
+# Firebase Database Schema
+![Diagram](Firebase%20Database.png)
```

Comment:
What's the source representation for this image?  I'm hesitant to use imported drawings for anything of substance because it's not really "source" and makes it harder to maintain in the long run. Really would like to have a textual representation of whatever it is so it can be managed w/ version control...